### ISSUE TITLE
Update to newer stable version of LAMMPS

### ISSUE THREAD

**Issue by nathanfranklin** [simphony/simphony-lammps-md#107]
We are currently using "r12824" from lammps (list of stable versions at http://lammps.sandia.gov/bug.html ).  However, the git tag is missing (removed?) in the last week.  In addition the install process for building the shared library has been changed.

**Comment by nathanfranklin** [comment]
@TobiasRasp

**Comment by nathanfranklin** [comment]
I was able to check out the 'r12824' version without a problem:
```pc-p561(~/local/temp) git clone --branch r12824 --depth 1 git://git.lammps.org/lammps-ro.git mylammps
Cloning into 'mylammps'...
remote: Counting objects: 7141, done.
remote: Compressing objects: 100% (5337/5337), done.
remote: Total 7141 (delta 2277), reused 4532 (delta 1740)
Receiving objects: 100% (7141/7141), 57.49 MiB | 881.00 KiB/s, done.
Resolving deltas: 100% (2277/2277), done.
Checking connectivity... done
Note: checking out 'bfb56a8918cb5690c5651aed62d5176544bd930a'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b new_branch_name

Checking out files: 100% (6821/6821), done.
```

I will still upgrade to the newest stable version.

**Comment by nathanfranklin** [comment]
I will update to the following (from http://lammps.sandia.gov/bug.html):

'10 Aug 2015 = stable version, SVN rev = 13864, Git tag = r13864 '

**Comment by nathanfranklin** [comment]
Maybe we should use the mirror (https://github.com/lammps/lammps) instead as I also just noticed how slow git.lammps.org is.

**Comment by nathanfranklin** [comment]
closed by #109
